cmd/zed/agents/zfs_agents.c

```diff
@@ -53,13 +53,20 @@ pthread_t g_agents_tid;
 libzfs_handle_t *g_zfs_hdl;
 
 /* guid search data */
+typedef enum device_type {
+	DEVICE_TYPE_L2ARC,
+	DEVICE_TYPE_SPARE,
+	DEVICE_TYPE_DISK
```

Comment:
What do you think about renaming `DEVICE_TYPE_DISK` to `DEVICE_TYPE_PRIMARY` (or similar).  What's being determined here is if the vdev is part of the primary pool storage.

Reply:
I couldn't think of a better name, i used `DEVICE_TYPE_DISK` (and `DEVICE_TYPE_L2ARC`/`DEVICE_TYPE_SPARE`) just because i found similarly named #defines here:

https://github.com/zfsonlinux/zfs/blob/54eb2c410ec8ef9f75c62e3d0e8f53ac8bdb5eea/include/sys/fs/zfs.h#L748-L754

`DEVICE_TYPE_PRIMARY` makes sense, i am going to use it.